test/Pester/PlatyPs.Tests.ps1

```diff
@@ -78,15 +85,17 @@ Describe 'New-PlatyPsCab' {
 
 Describe 'Format-PlatyPsHelpXml' {
 
-    $Destination = "$outFolder\CabTesting\OutXml\"
+    $Destination = "$outFolder\CabTesting\OutXml2\"
```

Comment:
@jongeller I changed a folder for this test, because without it test suite is failing if you run it twice. This 2 .xml files are in the same folder and assert logic in 'validates the output of Cab creation' (line 82) fails